### PR TITLE
Enhancement: toggle button read

### LINKED ISSUE #115: Toggle button for book read form control
Add book form has checkbox for book read input where both yes and no can be selected
Instead of checkbox there should be radio button or some sort of toggle button 
Can I work on this issue?

### PR DESCRIPTION
**Pull request by PratikshaManjrekar**
Added toggle button for read form section where user inputs whether book is read or not
Fix: #115

### CODE REVIEW

**Comment by samipak458** [comment]
Kindly add short video of your changes @PratikshaManjrekar

**Comment by PratikshaManjrekar** [comment]
https://user-images.githubusercontent.com/75800831/197353161-7247010a-8b79-433a-9eb4-e854d1343249.mp4

Added read toggle button

**Comment by samipak458** [comment]
Kindly sync the fork @PratikshaManjrekar

File: index.html [hunk 1]
```diff
@@ -134,12 +134,15 @@ <h3 class="text-center">
                 </div>
             </div>
             <div class="form-group row">
-                <label for="Author" class="col-sm-2">Read :</label>
-                <div class="col-sm-1">
-                    <input type="checkbox" id="read">Yes</input>
-                    <input type="checkbox" id="read">No</input>
+                <label for="Favorite" class="col-sm-2 col-form-label">Read:</label>
```
Comment (ghost) [inline index.html]: Putting the "favorite" and "read" switch button side by side makes it more accessible.

Comment (PratikshaManjrekar) [reply index.html]: Done

**Comment by PratikshaManjrekar** [comment]
> Kindly sync the fork @PratikshaManjrekar

Done

**Pull request merged**